### PR TITLE
Docs: update identity workflows

### PR DESCRIPTION
**Pull request by Eve832**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by nomeata** [comment]
@Eve832 A “Do not merge” label is good, but may be missed. A better way to indicate that a PR should not be merged is to make it a “draft”. You can do that while creating a PR, or, easy to miss, after the fact using a grey link in the top right corner.

**Comment by Eve832** [comment]
Or I can put DO NOT MERGE in the title, I guess.

**Comment by nomeata** [comment]
Sure but that won't disable the merge button, and assumes people read text - always a dangerous assumption :-D

File: docs/modules/ic-identity-guide/pages/auth-how-to.adoc
Comment (nomeata) [inline docs/modules/ic-identity-guide/pages/auth-how-to.adoc:76]:
```suggestion
Use a dedicated security key to recover your account in the event that you lose access to your authorized devices. This key must be different from one that you actively use to log in to the Identity service. Keep this key somewhere safe and known only to you. You will need to know your user number to begin recovery.
```

**Review by nomeata — APPROVED** [review]

**Pull request merged**
